Sources/SpeziFirebaseAccountStorage/SpeziFirebaseAccountStorage.docc/SpeziFirebaseAccountStorage.md

```diff
@@ -36,15 +36,36 @@ import SpeziFirebaseAccount
 import SpeziFirebaseAccountStorage
 
 class ExampleAppDelegate: SpeziAppDelegate {
-override var configuration: Configuration {
-    Configuration {
-        AccountConfiguration(
-            service: FirebaseAccountService(),
-            storageProvider: FirestoreAccountStorage(storeIn: Firestore.firestore().collection("users"))
-            configuration: [/* ... */]
-        )
+    override var configuration: Configuration {
+        Configuration {
+            AccountConfiguration(
+                service: FirebaseAccountService(),
+                storageProvider: FirestoreAccountStorage(storeIn: Firestore.firestore().collection("users")),
+                configuration: [/* ... */]
+            )
+        }
     }
 }
+
+### Custom Encoder/Decoder Configuration
+
+For advanced use cases, such as integrating with libraries like PhoneNumberKit where you might want to set specific encoding/decoding strategies,
+you can provide custom encoder and decoder instances with specific userInfo configurations.
+
+```swift
```

Comment:
Similar to the inline documentation, it might make sense to use an example that works in the Spezi `Configuration` setting to make it easier to reuse for new users to Spezi.